adm_program/system/classes/session.php

```diff
@@ -41,11 +44,16 @@ class Session extends TableAccess
      * @param int|string $session  The recordset of the session with this id will be loaded.
      *                             The session can be the table id or the alphanumeric session id.
      *                             If id isn't set than an empty object of the table is created.
+     * @param string     $cookiePrefix The prefix that is used for cookies
      */
-    public function __construct(&$database, $session = 0)
+    public function __construct(&$database, $session = 0, $cookiePrefix = '')
     {
+        global $gDb;
```

Comment:
yes, this was a test code because of a problem I had. I forgot to remove it after I resolve the problem :/